HPXMLtoOpenStudio/resources/battery.rb

```diff
@@ -73,6 +73,7 @@ def self.apply(runner, model, pv_systems, battery, schedules_file)
     elcs.setNumberofStringsinParallel(number_of_strings_in_parallel)
     elcs.setInitialFractionalStateofCharge(0.0)
     elcs.setBatteryMass(battery_mass)
+    elcs.setDCtoDCChargingEfficiency(0.925) #Based on Tesla Powerwall round trip efficiency (new)
```

Comment:
@jmaguire1 @joseph-robertson Seems like we should make this an optional input as it's already [a field in HPXML](https://hpxml.nrel.gov/datadictionary/3.0.0/Building/BuildingDetails/Systems/Batteries/Battery/RoundTripEfficiency).

Reply:
@shorowit: Agreed, would be better to take it as an input if we're already set up for that. @joseph-robertson offered to help me out with this one, I'm inclined to take him up on it.

Reply:
Works for me. Have at it, @joseph-robertson.